### PR TITLE
Add and implement category model

### PR DESCRIPTION
**Pull request by duyhoan**

### CODE REVIEW

File: app/views/categories/_category.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="panel panel-primary">
+  <div class="panel-body">
+    <div class="pull-left"><%= category.name %></div>
+    <div class="pull-right"><%= link_to "Start", categories_path, class: "btn btn-primary list-lessons" %></div>
```
Comment (dieunb) [inline app/views/categories/_category.html.erb]:
```
<div class="pull-right">
  <%= link_to "Start", categories_path, class: "btn btn-primary list-lessons" %>
</div>
```

File: app/helpers/categories_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module CategoriesHelper
```
Comment (dieunb) [inline app/helpers/categories_helper.rb]: remove

**Comment by dieunb** [comment]
OK

File: app/controllers/categories_controller.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class CategoriesController < ApplicationController
+
+  def index
+    @categories = Category.all
```
Comment (chuanhtuan) [inline app/controllers/categories_controller.rb]: paging???

**Pull request merged**